crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -723,9 +731,22 @@ fn timeout_in_5_min() -> u64 {
 
 #[cfg(test)]
 mod tests {
-    use super::max_timeout_height;
+    use super::*;
+
     #[test]
     fn max_timeout_height_does_not_panic() {
         max_timeout_height();
     }
+
+    #[test]
+    fn memo_snapshot() {
```

Comment:
bikshed: the fact that we have a `*_snapshot` suffixed file in the `snapshots/` directory is irking me. :D

Reply:
Renamed test to `memo_format_should_not_change` in [32b1522](https://github.com/astriaorg/astria/pull/1753/commits/32b15220e4e70a75cc134a150d1c0d3c9fdfd222).